lib/cylc/daemonize.py

```diff
@@ -70,7 +70,7 @@ def daemonize(server):
                     suite_pid is None or suite_url is None):
                 sleep(0.1)
                 try:
-                    # Line 1 (or 2 in debug mode) of suite log should contain
+                    # First INFO line of suite log should contain
```

Comment:
Out of interest, why did you decide against the contact file option here?

Reply:
Ultimately, we should go for the contact file option, but it involves a bit too much logic re-ordering that may cause other unexpected issues.

Reply:
Sounds like a good call.